docs/applications/requestrr.mdx

```diff
@@ -0,0 +1,137 @@
+10. Now copy the Bot Token
+11. Paste it into Requestrr in the Bot Token Section
+12. Click Test Settings button and it should say everything is good
+13. Click Save Changes ath the bottom of the page
+
+## Inviting Bot to the Server and Testing
+1. Copy the Invite Link
+2. Paste it into your browser and choose your Discord Server
+3. Test your new bot out by typing `ping!` in chat and it should respond `!pong`
+
+## Connecting to Other Applications
+
+If you would like to connect ombi to other applications you installed with swizzin, please follow the guides below.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="Radarr"
+  values={[
+    {label: 'Radarr', value: 'radarr'},
+    {label: 'Ombi', value: 'ombi'},
+    {label: 'Sonarr', value: 'sonarr'},
+    {label: 'Overseerr', value: 'overseerr'},
+  ]}>
+<TabItem value="radarr">
+
+
+  Go to **Settings** -> **General**
+  
+  1. Copy your API Key
+  2. In Requestrr, go to the "Movies Tab"
+  3. Paste the API Key into Requestrr API Key Field
+  4. Enter the hostname or IP, If Requestrr and Radarr are on the same machine then use `127.0.0.1`
+  5. Edit the "Radarr Movie Settings" as you see fit
+  6. Press Test Settings at the bottom of the page
+  7. Save your changes
+
+ </TabItem>
+
+ <TabItem value="ombi">
+
+
+  Go to **Settings** -> **Ombi**
+  
+  1. Copy your API Key
+  2. In Requestrr, go to the "Movies Tab/TV Shows Tab"
+  3. Paste the API Key into Requestrr API Key Field
+  4. Enter the hostname or IP, If Requestrr and Ombi are on the same machine then use `127.0.0.1`
+  5. Press Test Settings at the bottom of the page
+  6. Save your changes
+
+More information in regards to permissions and linking a Discord user to an Ombi user can be found [Here](https://github.com/darkalfx/requestrr/wiki/Configuring-Ombi#configuring-permissions)
+
+ </TabItem>
+
+<TabItem value="sonarr">
+
+
+  Go to **Settings** -> **General**
+  
+  1. Copy your API Key
+  2. In Requestrr, go to the "TV Shows Tab"
+  3. Paste the API Key into Requestrr API Key Field
+  4. Enter the hostname or IP, If Requestrr and Sonarr are on the same machine then use `127.0.0.1`
+  5. Edit the "Sonarr TV Show Settings" as you see fit
+  6. Press Test Settings at the bottom of the page
+  7. Save your changes
+
+ </TabItem>
+
+<TabItem value="overseerr">
+
+
+  Go to **Settings** -> **General Settings**
+  
+  1. Make sure CSRF protection is disabled
+  2. Still in the Overseerr settings page, copy the API Key
+  3. Paste the API Key into Requestrr API Key Field
+  4. Enter the hostname or IP, If Requestrr and Radarr are on the same machine then use `127.0.0.1`
+  5. Press Test Settings at the bottom of the page
+  6. Save your changes
+
+More information in regards to permissions and linking a Discord user to an Overseerr user can be found [Here](https://github.com/darkalfx/requestrr/wiki/Configuring-Overseerr#configuring-permissions)
+
+ </TabItem>
+
```

Comment:
you need to close tabs. see this for an example: 
https://pipelines.actions.githubusercontent.com/etBdarlm6eMDwSa1NkNZo2fsgjLVPIg6bOIIEVHCrEhB8zbABn/_apis/pipelines/1/runs/153/signedlogcontent/4?urlExpires=2021-03-22T04%3A33%3A59.4675604Z&urlSigningMethod=HMACV1&urlSignature=5qZUA7K5FB0aFQlBVY7aw6IzQhf3cr%2F8ZeTGcCdTCqw%3D